src/main/java/com/google/flourbot/execution/MacroExecutionModuleImplementation.java

```diff
@@ -134,23 +137,31 @@ public String execute(String userEmail, String message, String threadId) throws
 
         // Append values to first free bottom row of sheet
         cloudSheet.appendRow(values);
-        replyText = "Appended row to " + action.getSheetUrl();
+        chatResponse = ChatResponse.createChatResponseWithList(values, actionType, documentUrl);
         break;
+
       case SHEET_READ_ROW:
         List<String> rowData = cloudSheet.readRow(2); //TODO: Replace hardcoded
-        replyText = rowData.get(0);
+        chatResponse = ChatResponse.createChatResponseWithList(rowData, actionType, documentUrl);
         break;
+
       case SHEET_READ_COLUMN:
         List<String> columnData = cloudSheet.readColumn("B");//TODO: Replace hardcoded
-        replyText = columnData.get(0);
+        chatResponse = ChatResponse.createChatResponseWithList(columnData, actionType, documentUrl);
+        break;
+
+      case SHEET_READ_SHEET:
+        List<List<String>> sheetData = cloudSheet.readSheet("Sheet1");
```

Comment:
I guess this can be our default, but it would be good to give the user a chance to define the sheet name when the macro is created.

Reply:
Yes! My only concern is that the user can edit the sheet name afterwards and we won't see the change. We would have to add more logic to find all the sheetnames in the Sheets doc and I don't know how we would know to select the proper one. So it seems simpler to tell the user to keep the default name

Reply:
At least for our goals, we can come back to this if time permits